src/OrchardCore.Modules/OrchardCore.Deployment/Assets/js/steporder.js

```diff
@@ -19,13 +19,14 @@ function updateStepOrders(oldIndex, newIndex) {
 
 $(function () {
     var sortable = document.getElementById("stepOrder");
-
-    var sortable = Sortable.create(sortable, {
-        handle: ".ui-sortable-handle",
-        onSort: function (evt) {
-            var oldIndex = evt.oldIndex;
-            var newIndex = evt.newIndex;
-            updateStepOrders(oldIndex, newIndex);
-        }
-    });
+    if (sortable) {
```

Comment:
I like to return `if (!sortable)` to save indentation.

Also, please eliminate the need for jQuery here by replacing $(function()... to

`document.addEventHandler('DOMContentLoaded', function()`

Reply:
I second the indentation, but as for jQuery: we already use it everywhere on the admin, and this script uses it 6 times too. I thus wouldn't change the ready function and risk breaking something in some obscure way when we wouldn't really get any benefit (and especially don't scope creep into trying to get rid of jQuery too).